defaults/main.yml

```diff
@@ -15,8 +15,13 @@ apache_remove_default_vhost: false
 apache_global_vhost_settings: |
   DirectoryIndex index.php index.html
 
+apache_htpasswd:
+  # - file: "/var/www/.htpasswd"
+  #   username: "myuser"
+  #   password: "mypassword"
```

Comment:
For the passwords, is it common/okay to store them as plaintext in these files? I thought it was best to at least perform some sort of hashing prior to storage in the .htpasswd file.

Also, it seems this would only provide one user/pass pair per file—isn't it normal/common to have multiple in a file?